app/services/package-type-service.js

```diff
@@ -41,10 +41,6 @@ export default Ember.Service.extend({
     return deferred.promise;
   },
 
-  preload: cached(async function() {
-    return await this.get("store").query("code", { stock: true });
-  }),
-
```

Comment:
I believe this is used in the Stocktake route. Please make a search on the whole project and make sure there's no call to that function